src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import type { ReportHandler } from 'web-vitals';
```

Comment:
I think we could remove this part for now

Reply:
like remove the whole file?

Reply:
Yes, exactly. I did it.